### PR TITLE
V4.0 Documents Page update

### PR DESCRIPTION
**Pull request by mollybehan**
- Created a README.md table on the documents page to help clarify what resources are available.
- Added several documents collected by the TA Co-chairs to serve as references for policy language and data security.

### CODE REVIEW

File: documents/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+| Document Title | Publisher | Description |
```
Comment (j-d-b) [inline documents/README.md:1]: I think you should have line 1 be a header, for example:

```
# Documents
```

Then the table below. Otherwise looks good!

Comment (mollybehan) [reply documents/README.md:2]: Thanks for the feedback, I've added the header.

File: documents/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Documents
+| Document Title | Publisher | Description |
```
Comment (j-d-b) [inline documents/README.md:2]: FYI this table markdown can be simpler. You don't need the `|` on the outside of each row, nor as many `------`. It keeps the file smaller and easier to read when not rendered. It is also then consistent with how we represent tables in other markdown files.

For example:

```
Document Title | Publisher | Description
--- | --- | ---
this | row | content
```

**Review by j-d-b — APPROVED** [review]
Thanks for making the changes. Approved: you can merge the PR any time.

**Pull request merged**